### PR TITLE
Make doit mentions in map key be hyperlinks

### PR DESCRIPTION
**Pull request by nnarh**
https://www.pivotaltracker.com/n/projects/742821/stories/122461371

### CODE REVIEW

**Comment by tansaku** [comment]
Hi Nick - this looks great - want to fix up those hound issues and replace the quotes with singles?  If not convenient for you I can do myself - otherwise I think this is good to go :-)

**Comment by johnnymo87** [comment]
Nick and I were discussing this last night -- Github's new UI unfortunately makes it look like we didn't address these comments from hound, even though we did.

File: features/step_definitions/volunteer_op_steps.rb [hunk 1]
```diff
@@ -95,3 +95,17 @@
...
   expect { click_link(organisation) }.to change(&number_of_windows).by 1
 end
+
+Then(/^the Do-it word in the legend should be a hyperlink to the Do-it website$/) do
+  within('.map_legend') do
+    first, _, third = all('.key_text').to_a
+
+    expect(
+      first.has_link?('Do-it', href: 'https://do-it.org/')
+    ).to eq(true)
```
Comment (tansaku) [inline features/step_definitions/volunteer_op_steps.rb]: could replace with:

```
expect(first).to have_link('Do-it', href: 'https://do-it.org/')
```

Comment (nnarh) [inline features/step_definitions/volunteer_op_steps.rb]: I like this. done

**Comment by tansaku** [comment]
great - thanks @nnarh !  

I can see room for further refactoring and Drying Out in:

```ruby
Then(/^the Do-it word in the legend should be a hyperlink to the Do-it website$/) do
  within('.map_legend') do
    first, _, third = all('.key_text').to_a

    expect(first).to have_link('Do-it', href: 'https://do-it.org/')
    expect(third).to have_link('Do-it', href: 'https://do-it.org/')
  end
end
```

where we avoid this breaking in the face of changing orderings in the map legend, but that could be a separate refactoring ticket if anyone was interested.

In the meantime I'll merge this.  Thanks for your contribution!

**Pull request merged**